kitsune/sumo/static/sumo/js/form-wizard.js

```diff
@@ -34,6 +55,24 @@ export class FormWizard extends HTMLElement {
   set activeStep(name) {
     this.#activeStep = name;
     this.#setActiveStepAttributes();
+    this.#updateFormProgress();
+  }
+
+  get #steps() {
+    return [...this.children].map((child) => child.getAttribute("name"));
+  }
+
+  /**
+   * Update the value of the progress element to show form completion.
+   * Defaults to 10% for the first step.
+   */
+  #updateFormProgress() {
+    if (this.#steps?.length) {
+      let activeStepIndex = this.#steps.indexOf(this.activeStep);
```

Comment:
Yeah I almost immediately started tripping over this when adding the step indicator. I've changed `#steps` to a private field and added a `MutationObserver` to keep things in sync, just in case.